spec/features/idv/steps/in_person/address_spec.rb

```diff
@@ -55,6 +56,143 @@
       expect(page).to have_text(InPersonHelper::GOOD_ADDRESS2)
       expect(page).to have_text(InPersonHelper::GOOD_CITY)
       expect(page).to have_text(InPersonHelper::GOOD_ZIPCODE)
+      expect(page).to have_text(Idp::Constants::MOCK_IDV_APPLICANT_STATE)
```

Comment:
Is this a new field that we weren't checking for before or was this a wee oversight?

Reply:
I think it was an oversight as it existed on the page previously but we werent checking for it in the spec